packages/cli/test/Prompt.test.ts

```diff
@@ -0,0 +1,84 @@
+import type * as CliApp from "@effect/cli/CliApp"
+import * as Command from "@effect/cli/Command"
+import * as Prompt from "@effect/cli/Prompt"
+import * as MockConsole from "@effect/cli/test/services/MockConsole"
+import * as MockTerminal from "@effect/cli/test/services/MockTerminal"
+import {} from "@effect/platform"
+import { Array, Effect } from "effect"
+import * as Console from "effect/Console"
+import * as Fiber from "effect/Fiber"
+import * as Layer from "effect/Layer"
+import { describe, expect, it } from "vitest"
+
+const MainLive = Effect.gen(function*(_) {
+  const console = yield* _(MockConsole.make)
+  return Layer.mergeAll(
+    Console.setConsole(console),
+    MockTerminal.layer
+  )
+}).pipe(Layer.unwrapEffect)
+
+const runEffect = <E, A>(
+  self: Effect.Effect<A, E, CliApp.CliApp.Environment>
+): Promise<A> => Effect.provide(self, MainLive).pipe(Effect.runPromise)
+
+describe("Prompt", () => {
+  describe("text", () => {
+    it("should return an empty string when `default` on `Prompt.TextOptions` is not provided", () =>
+      Effect.gen(function*(_) {
```

Comment:
```suggestion
      Effect.gen(function*() {
```

Given that the adapter in `Effect.gen` is no longer required, we are trying to avoid introducing new code into the codebase that makes use of it.